desktop/libs/azure/src/azure/abfs/abfs.py

```diff
@@ -338,11 +345,13 @@ def append(self, path, data, size = 0, offset =0 ,params = None, **kwargs):
     else:
       params['action'] = 'append'
     headers = {}
-    if size == 0:
+    if size == 0 or size == '0':
```

Comment:
When is size a string?

Reply:
The header 'Content-Length' is supposed to be passed as an int, but ABFS runs into an error saying I should pass the size as a string. So all ints should be converted to strings. Also, Python doesn't check types until runtime, it's possible for the user or another programer to pass a string instead of an int as the parameter for size.